apps/sv/frontend/src/hooks/useProposalMutation.tsx

```diff
@@ -51,7 +51,6 @@ export const useProposalMutation: () => UseMutationResult<
 
     onError: error => {
       console.error(`Failed to send proposal to dso`, error);
-      toast.error(`Failed to send proposal to dso: ${error.message}`);
```

Comment:
We agreed to now show a toast on error in the last meeting so I'm removing it here.